src/servers/src/http/header.rs

```diff
@@ -17,7 +17,7 @@ use headers::{Header, HeaderName, HeaderValue};
 pub const GREPTIME_DB_HEADER_FORMAT: &str = "x-greptime-format";
 pub const GREPTIME_DB_HEADER_EXECUTION_TIME: &str = "x-greptime-execution-time";
 
-pub static GREPTIME_DB_HEADER_NAME: HeaderName = HeaderName::from_static("x-greptime-name");
+pub static GREPTIME_DB_HEADER_NAME: HeaderName = HeaderName::from_static("x-greptime-db-name");
```

Comment:
Adds a comment for these constants to remind developers not to modify.

Reply:
We're planning to start a container to send request to our greptimedb cluster and check results, much like sqlness but from a separate outside pod, everytime our cluster is deployed. It should serve as independent integration and compatibility test.  cc @sunng87